code/game/machinery/autolathe_datums.dm

```diff
@@ -456,3 +456,55 @@
 	name = "miniature cell"
 	path = /obj/item/weapon/cell/device/emergency_light/empty
 	category = "Engineering"
+	
+/datum/autolathe/recipe/cable_coil
+	name = "cable coil"
+	path = /obj/item/stack/cable_coil
+	category = "Devices and Components"
+	is_stack = 1
+
+/datum/autolathe/recipe/labeler
+	name = "hand labeler"
+	path = /obj/item/weapon/hand_labeler
+	category = "General"
+
+/datum/autolathe/recipe/destTagger
+	name = "destination tagger"
+	path = /obj/item/device/destTagger
+	category = "General"
+
+/datum/autolathe/recipe/implanter
+	name = "implanter"
+	path = /obj/item/weapon/implanter
+	hidden = 1
+	category = "Medical"
+	
+/datum/autolathe/recipe/debugger
+	name = "debugger"
+	path = /obj/item/device/debugger
+	category = "Engineering"
+	
+/datum/autolathe/recipe/micromanip
```

Comment:
This is included in the protolathe, no need for the autolathe to have it.

Reply:
The protolathe only makes circuit debuggers not the engineering ones (the ones that fight blue APCs) as far as I'm aware, this was an issue that I heard about when engineering needed them to fight a malf AI